docs/reference/api/types.md

```diff
@@ -13,3 +13,30 @@ A few types used in Metaxy here and there.
 ::: metaxy.models.types.SnapshotPushResult
 
 ::: metaxy.IDColumns
+
+## Narwhals Filter Expressions
+
+Narwhals filter expressions let you serialize and deserialize backend-agnostic predicates.
+They are backed by SQLGlot and understand the following subset of SQL `WHERE` syntax:
+
+- Comparisons: `=`, `!=`, `>`, `<`, `>=`, `<=`
```

Comment:
I think this list is missing new lines - are you rendering the docs for yourself?